build.gradle

```diff
@@ -14,7 +14,7 @@ plugins {
 
 group = 'software.amazon.cryptools'
 version = '2.5.0'
-ext.awsLcMainTag = 'v1.60.0'
+ext.awsLcMainTag = 'xaes-256-gcm'
```

Comment:
this is fine for the purposes of this PR while XAES changes remain on a non-mainline AWS-LC branch.

I've converted this PR to "draft" status until/if upstream XAES changes are merged in AWS-LC.